crates/youki/src/logger.rs

```diff
@@ -66,47 +74,94 @@ fn detect_log_level(is_debug: bool) -> Result<LevelFilter> {
     Ok(LevelFilter::from_str(filter.as_ref())?)
 }
 
-fn json_write<F: 'static>(f: &mut F, record: &log::Record) -> std::io::Result<()>
-where
-    F: Write,
-{
-    write!(f, "{{")?;
-    write!(f, "\"level\":\"{}\",", record.level())?;
-    write!(f, "\"time\":\"{}\"", chrono::Local::now().to_rfc3339(),)?;
-    write!(f, ",\"msg\":")?;
-    // Use serde_json here so we don't have to worry about escaping special characters in the string.
-    serde_json::to_writer(f.by_ref(), &record.args().to_string())?;
-    writeln!(f, "}}")?;
+struct YoukiLogger {
+    /// Indicates level up to which logs are to be printed
+    level: Option<log::Level>,
```

Comment:
What is the case of `None`?

Reply:
None means logging is disabled. None is returned by to_level if a logging level of 'Off' is specified.

Reply:
I forgot about it! You're right.